third_party/terraform/data_sources/data_source_google_app_engine_default_service_account.go

```diff
@@ -0,0 +1,76 @@
+package google
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceGoogleAppEngineDefaultServiceAccount() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceGoogleAppEngineDefaultServiceAccountRead,
+		Schema: map[string]*schema.Schema{
+			"project": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"email": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"unique_id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"display_name": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+		},
+	}
+}
+
+func dataSourceGoogleAppEngineDefaultServiceAccountRead(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*Config)
+
+	project, err := getProject(d, config)
+	if err != nil {
+		return err
+	}
+
+	serviceAccountEmail := fmt.Sprintf("%s@appspot.gserviceaccount.com", project)
+
+	serviceAccountName, err := serviceAccountFQN(serviceAccountEmail, d, config)
+	if err != nil {
+		return err
+	}
+
+	sa, err := config.clientIAM.Projects.ServiceAccounts.Get(serviceAccountName).Do()
```

Comment:
```
google-beta/data_source_google_app_engine_default_service_account.go:53:19: config.clientIAM undefined (type *Config has no field or method clientIAM)
note: module requires Go 1.14
make: *** [build] Error 2
GNUmakefile:9: recipe for target 'build' failed
```

There was a pretty recent change made to config.go so I'd recommend rebasing, making the necessary changes, and then running `make` in the downstreams again.

Reply:
I just ran in to that issue with the sql delete protection PR. I'll get it fixed.

Reply:
fixed